Content.Server/Nutrition/Components/FoodComponent.cs

```diff
@@ -6,87 +6,94 @@
 
-namespace Content.Server.Nutrition.Components
+namespace Content.Server.Nutrition.Components;
+
+[RegisterComponent, Access(typeof(FoodSystem))]
+public sealed partial class FoodComponent : Component
 {
-    [RegisterComponent, Access(typeof(FoodSystem))]
-    public sealed partial class FoodComponent : Component
+    [DataField("solution")]
+    public string SolutionName = "food";
+
+    [DataField("useSound")]
+    public SoundSpecifier UseSound = new SoundPathSpecifier("/Audio/Items/eatfood.ogg");
+
+    [DataField("trash", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string? TrashPrototype;
+
+    [DataField("transferAmount")]
+    public FixedPoint2? TransferAmount = FixedPoint2.New(5);
+
+    /// <summary>
+    /// Acceptable utensil to use
+    /// </summary>
+    [DataField("utensil")]
+    public UtensilType Utensil = UtensilType.Fork; //There are more "solid" than "liquid" food
+
+    /// <summary>
+    /// Is utensil required to eat this food
+    /// </summary>
+    [DataField("utensilRequired")]
+    public bool UtensilRequired;
+
+    /// <summary>
+    ///     If this is set to true, food can only be eaten if you have a stomach with a
+    ///     <see cref="StomachComponent.SpecialDigestible"/> that includes this entity in its whitelist,
+    ///     rather than just being digestible by anything that can eat food.
+    ///     Whitelist the food component to allow eating of normal food.
+    /// </summary>
+    [DataField("requiresSpecialDigestion")]
+    public bool RequiresSpecialDigestion;
+
+    /// <summary>
+    ///     Stomachs required to digest this entity.
+    ///     Used to simulate 'ruminant' digestive systems (which can digest grass)
+    /// </summary>
+    [DataField("requiredStomachs")]
+    public int RequiredStomachs = 1;
+
+    /// <summary>
+    /// The localization identifier for the eat message. Needs a "food" entity argument passed to it.
+    /// </summary>
+    [DataField("eatMessage")]
+    public string EatMessage = "food-nom";
+
+    /// <summary>
+    /// The locale id for the popup shown when the food is openable but closed. Needs a "owner" entity argument passed to it.
+    /// Defaults to the popup drink uses since its "correct".
+    /// It's still generic enough that you should change it if you make openable food, i.e. unwrap it first, peel it first.
+    /// </summary>
+    [DataField("closedPopup")]
+    public string ClosedPopup = "drink-component-try-use-drink-not-open";
+
+    /// <summary>
+    /// How long it takes to eat the food personally.
+    /// </summary>
+    [DataField("delay")]
+    public float Delay = 1;
+
+    /// <summary>
+    ///     This is how many seconds it takes to force feed someone this food.
+    ///     Should probably be smaller for small items like pills.
+    /// </summary>
+    [DataField("forceFeedDelay")]
+    public float ForceFeedDelay = 3;
+
+    [ViewVariables]
```

Comment:
ReadWrite

Reply:
it only has get, set would be editing solutions which can be done less hackily in solution editor just add nutriment or vitamin or whatever you want

Reply:
oh removed the field anyway